atomic_reactor/plugins/pre_flatpak_update_dockerfile.py

```diff
@@ -244,18 +204,43 @@ def run(self):
 
         dockerfile.content = content
 
+    def create_includepkgs_file_and_cleanupscript(self, builder, build_dir: BuildDir):
         # Create a file describing which packages from the base yum repositories are included
-
         includepkgs = builder.get_includepkgs()
-        includepkgs_path = os.path.join(self.workflow.df_dir,
+        includepkgs_path = os.path.join(build_dir.path,
                                         FLATPAK_INCLUDEPKGS_FILENAME)
         with open(includepkgs_path, 'w') as f:
             f.write('includepkgs = ' + ','.join(includepkgs) + '\n')
 
         # Create the cleanup script
-
-        cleanupscript = os.path.join(self.workflow.df_dir,
+        cleanupscript = os.path.join(build_dir.path,
                                      FLATPAK_CLEANUPSCRIPT_FILENAME)
         with open(cleanupscript, 'w') as f:
             f.write(builder.get_cleanup_script())
         os.chmod(cleanupscript, 0o0500)
+        return [Path(includepkgs_path), Path(cleanupscript)]
```

Comment:
You could take full advantage of Path objects like this:
```python
cleanupscript = build_dir.path / FLATPAK_CLEANUPSCRIPT_FILENAME
cleanupscript.write_text(builder.get_cleanup_script())
cleanupscript.chmod(0o0500)
return [includepkgs_path, cleanupscript]
```
(and similarly for the includepkg_path)

Completely optional, I just happen to be a fan of [pathlib](https://docs.python.org/3/library/pathlib.html) :smile: